### PR TITLE
Added UI and updated class component with functional component and hooks

### PR DESCRIPTION
**Pull request by dnyanesh-bhalerao**

### CODE REVIEW

File: src/containers/Demo/AccordionList.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { Accordion, Button, Card } from 'react-bootstrap';
+
+interface IAccordionProps {
+  listData: DemoData[];
+}
+
+const AccordionList: React.FC<IAccordionProps> = ({ listData }) => {
+  return (
+    <Accordion>
+      {listData.map(listItem => (
```
Comment (omkar-joshi) [inline src/containers/Demo/AccordionList.tsx]: Move this to a separate function - `renderList`

Comment (omkar-joshi) [reply src/containers/Demo/AccordionList.tsx]:
```
return <Accordion>{renderList()}</Accordion>
```

File: src/containers/Demo/index.tsx [hunk 2]
```diff
@@ -1,57 +1,37 @@
...
-  loading: boolean;
-  mockPosts: DemoData[];
-}
+  const dispatch = useDispatch();
 
-type IDemoProps = IDemoDispatchProps & IDemoStateProps;
+  useEffect(() => {
+    dispatch(getDemoData());
+    dispatch(getMockData());
+    /* tslint:disable-next-line */
```
Comment (omkar-joshi) [inline src/containers/Demo/index.tsx:20]: Why do we need this? We are not using any such rule

**Review by omkar-joshi — CHANGES_REQUESTED** [review]

File: src/containers/Demo/AccordionList.tsx (hunk 1 above)
Comment (dnyanesh-bhalerao) [reply src/containers/Demo/AccordionList.tsx]: Done

File: src/containers/Demo/index.tsx (hunk 2 above)
Comment (dnyanesh-bhalerao) [reply src/containers/Demo/index.tsx:20]: We have added alignment rule for arguments in tslint.json, which results in `arguments are not aligned` error.

**Review by omkar-joshi — APPROVED** [review]

**Pull request merged**